### ISSUE TITLE
Request for a Font Like Non-mechanical Krypton

### ISSUE THREAD

**Issue by LambdaAlpha** [githubnext/monaspace#175]
I really like the idea of font mixing and texture healing. But after a week of trial, I found that I'm not quite fond of the styles of the existing five fonts. I dislike the presence of unnecessary lines in the fonts, such as the underlines in `i`, `l`, `f`, `r`. Unfortunately, only the Krypton font lacks these underlines, but I'm not a fan of its mechanical style and the jagged lines. I used to use the Source Code Pro font, and I really liked its straightforward and clean style. Can Monaspace create a font with a similar style? I would greatly appreciate it!

**Comment by yvvt0379** [comment]
Agree. I've been using JetBrains Mono for quite a long time and I really find it clean. There are no extra details in each character, which makes code reading quite smooth. 

But when I try using Mona, I find some of the letters too jarring. There are too many unnecessary details, they catch my eyes and disturb code reading.

**Comment by tianfanghan** [comment]
> Agree. I've been using JetBrains Mono for quite a long time and I really find it clean. There are no extra details in each character, which makes code reading quite smooth.
> 
> But when I try using Mona, I find some of the letters too jarring. There are too many unnecessary details, they catch my eyes and disturb code reading.

Agree with you.

**Comment by tianfanghan** [comment]
And name the New font "Helium".

**Comment by idan** [comment]
Sorry folks, I don't think a whole different font is in the cards.

That said, we've added `cv10` and `cv11`:

- `cv10`: `l` `i` alternates (Neon, Argon, Xenon, Radon)
- `cv11`: `j` `f` `r` `t` alternates (Neon, Argon)

These character variants supply alternate letterforms that might be what you're looking for!
